src/nih_wayfinding/app/scripts/views/navigate/navigate-controller.js

```diff
@@ -156,8 +156,7 @@
                   map.setZoom(13);
                 });
                 Notifications.show({
-                    text: 'Select a the destination you\'d like to be routed to',
-                    timeout: 3000
+                    text: 'Select a the destination you\'d like to be routed to'
```

Comment:
OH MAN! That was the one I originally saw that made me make the changes I did, but then couldn't find again.